README.md

```diff
@@ -68,72 +136,23 @@ Example command for DataflowRunner:
 
 ```bash
 python -m gcp_variant_transforms.vcf_to_bq \
-  --input_pattern gs://bucket/vcfs/vcffile.vcf \
-  --output_table projectname:bigquerydataset.tablename  \
-  --project projectname \
-  --staging_location gs://bucket/staging \
-  --temp_location gs://bucket/temp \
-  --job_name vcf-to-bq \
+  --input_pattern gs://my-bucket/vcffiles/*.vcf \
+  --output_table my-project:my-bigquery-dataset.my-bigquery-table \
+  --project my-project \
+  --staging_location gs://my-bucket/staging \
+  --temp_location gs://my-bucket/temp \
+  --job_name vcf-to-bigquery \
```

Comment:
Placing "<>" around parameters that need to be replaced can be helpful to users.

Reply:
See above.